src/components/themed/ShareButtons.js

```diff
@@ -0,0 +1,64 @@
+// @flow
```

Comment:
Are you sure you can't reuse any of the buttons we have in the common themed buttons file?

https://github.com/EdgeApp/edge-react-gui/blob/develop/src/components/themed/ThemedButtons.js

Reply:
Pretty sure, we already read every single components on there